### PR TITLE
Sap system link cluster details

### PR DESCRIPTION
**Pull request by arbulu89**
# Description

Add link to the sap system/database from the cluster details view using the SID.

Have a look on storybook if you want to see if life

image
image

## How was this tested?
Tested

### CODE REVIEW

File: assets/js/components/ClusterDetails/SapSystemLink.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```
Comment (arbulu89) [inline assets/js/components/ClusterDetails/SapSystemLink.jsx:1]: Didn't add an explicit test, as it is tested way better in conjunction with the main views, to have the whole context

**Review by nelsonkopliku — COMMENTED** [review]
Looks great!
I only have a doubt on the new component.

**Review by nelsonkopliku — APPROVED** [review]

**Review by dottorblaster — APPROVED** [review]
I don't know if I'm that enthusiastic of this usage of link components as they spread our need for router-enabled tests and stories a lot, but I don't really see a viable alternative :smile: 

LGTM

**Review by EMaksy — APPROVED** [review]
Great job, LGTM :D

**Pull request merged**